ls-ui-react-app/src/apiResponses.ts

```diff
@@ -30,10 +30,10 @@ class CompilerError {
 
 class CompilationResultSuccess {
   slices: Slices
-  compilationText: string
-  constructor(slices : Slices, compilationText : string) {
+  compilation_text: string
```

Comment:
I see, updating to match the format in the JSON. Hope this wasn't too tricky to debug :sweat_smile:, it was set that way just because it was using python's convention

Reply:
Initially I thought that the compilation text wasnt being returned by the API at all. It appears to me that to use the preferred type naming convetion compilationText, we would have to change the output of the API to match? Or have another intermediate assignment of apiResponse in the browser... but that just felt redundant. Hence why I just opted to match the attribute on the returned Json.

Reply:
Yea we would need to change the API ... it's good like this